onadata/apps/logger/tests/models/test_instance.py

```diff
@@ -30,7 +43,7 @@ def test_stores_json(self):
 
     @patch('django.utils.timezone.now')
     def test_json_assigns_attributes(self, mock_time):
-        mock_time.return_value = datetime.utcnow().replace(tzinfo=utc)
+        mock_time.return_value = timezone.datetime.now(timezone.utc)
```

Comment:
These changes are unrelated to the other work in this PR, but I was trying to clean up my imports and our use of utcnow() was marked as deprecated, so I figured I would fix it while I was there.